proposed/http-handlers/request-handlers.md

```diff
@@ -112,8 +112,8 @@ use Psr\Http\Message\ServerRequestInterface;
 /**
  * Participant in processing a server request and response
  *
- * An HTTP middleware component participates in processing an HTTP message,
- * either by acting on the request, generating the response, or forwarding the
+ * An HTTP middleware component participates in processing an HTTP message;
+ * for example, by acting on the request, generating the response, or forwarding the
```

Comment:
Why the change of phrasing? "Either" seems like the appropriate wording.

Reply:
This was based on feedback from @mnapoli, IIRC.

Reply:
This seems fine.  If we want to get super pedantic then "either" is technically only correct when there's 2 options, and in this case there are 3.  "For example" seems OK to me.

Reply:
Or, technically, with the semicolon we could omit that word entirely and it would still be a grammatically valid list of the available options.